vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/impl/GraphQLHandlerImpl.java

```diff
@@ -193,6 +196,9 @@ private void handlePost(RoutingContext rc, Buffer body) {
       case "application/json":
         handlePostJson(rc, body, rc.queryParams().get("operationName"), variables);
         break;
+      case "multipart/form-data":
+        handlePostMultipart(rc, rc.queryParams().get("operationName"), variables);
```

Comment:
This feature should be disabled by default. Can you a new entry to the handler options and check its value here?

Reply:
Done